src/guides/v2.3/howdoi/admin/customize_admin.md

```diff
@@ -0,0 +1,233 @@
+
+![Default Store View]({{ site.baseurl }}/common/images/design_conf2.png)
+
+Both the grid and the configuration form are implemented using UI components.
+
+To change the grid view, you need to [customize the grid](#customize-the-grid) configuration by adding a custom `design_config_listing.xml` to your module.
+
+To change the available design settings you need to [customize the grid](#customize-the-grid) configuration by adding your custom `design_config_form.xml` in your module. If you add a new field, you must also declare it in `di.xml` on how it is processed and saved.
+
+## Customize the grid
+The grid containing the configuration scopes is implemented using the [grid UI component]({{page.baseurl}}/ui_comp_guide/components/ui-listing-grid.html).
+
+To customize the grid view, take the following steps:
+
+1. In the `<your_module_dir>/view/adminhtml/ui_component` directory, add the empty `design_config_listing.xml`.
+
+1. In the `design_config_listing.xml` file, create an element to which to add your customizations. For example, if you want to rename the column displaying the selected theme, your grid configuration must contain the following:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+
+<listing xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
+    <columns name="design_config_columns">
+        <column name="theme_theme_id">
+            <argument name="data" xsi:type="array">
+                <item name="config" xsi:type="array">
+                    <item name="label" xsi:type="string" translate="true">%New theme column name%</item>
+                </item>
+            </argument>
+        </column>
+    </columns>
+</listing>
+```
+
+Your `design_config_listing.xml` is merged with the same files from the other modules. So there is no need to copy their content, you only need to define changes. Even if you want to customize the existing entities, you only have to mention those options for which the values are customized.
+
+For reference, view the grid configuration files of the Magento modules:
+
+-  `<Magento_Backend_module_dir>/view/adminhtml/ui_component/design_config_listing.xml`
+-  `<Magento_Theme_module_dir>/view/adminhtml/ui_component/design_config_listing.xml`
+
+If you add a certain field as an additional grid column, you also need to set the field’s `use_in_grid property` in the [field’s meta data](#add-fields-metadata) in `di.xml`.
+
+## Customize the design options
+
+These sections detail how to customize form configuration and field metadata.
+
+## Customize the form configuration
+
+The design configuration form is implemented using the form UI component.
+
+To customize the form view, take the following steps:
+
+1. Create an empty `design_config_form.xml` file in the `<your_module_dir>/view/adminhtml/ui_component/` directory.
+
+1. Add content similar to the following:
+
+```xml
```

Comment:
This entire code block must be indented three spaces to properly align under the preceding list number.

Reply:
Resolved